src/Components/SignUpForm/SignUpForm.js

```diff
@@ -1,14 +1,23 @@
-import React from 'react'
+import { Component} from 'react'
 import SignIn from '../../Containers/SignIn/SignIn'
 import SignUp from '../../Containers/SignUp/SignUp'
 
-const SignUpForm = () => {
+class SignUpForm extends Component {
+  constru() {
+    super();
+    this.state = {
+      signIn: false,
+      signUp: true
+    }
+  }
+  render() {
   return (
     <div>
```

Comment:
maybe we could find a more semantic option like section or an aside instead of a div